features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/listing/js/samples.js

```diff
@@ -1,9 +1,9 @@
-function APISamples () {
-    this.sample1_swagger = "{\"paths\":{ \"/*\":{\"get\":{\"parameters\":[{\"description\":\"Name of the City\",\"name\":\"q\",\"type\":\"string\",\"required\":false,\"in\":\"query\"}],\"responses\":{\"200\":{ } },\"x-auth-type\":\"Application & Application User\", \"x-throttling-tier\":\"Unlimited\" }}},\"swagger\":\"2.0\",\"info\":{ \"title\":\"WeatherAPI\",\"version\":\"1.0.0\"}}";
+function APISamples (defaultTier) {
+    this.sample1_swagger = "{\"paths\":{ \"/*\":{\"get\":{\"parameters\":[{\"description\":\"Name of the City\",\"name\":\"q\",\"type\":\"string\",\"required\":false,\"in\":\"query\"}],\"responses\":{\"200\":{ } },\"x-auth-type\":\"Application & Application User\", \"x-throttling-tier\":\"" + defaultTier + "\" }}},\"swagger\":\"2.0\",\"info\":{ \"title\":\"WeatherAPI\",\"version\":\"1.0.0\"}}";
 }
 
 
-APISamples.prototype.deploySample1 = function(){
+APISamples.prototype.deploySample1 = function(defaultTier){
```

Comment:
Please note that formatting issue has been corrected in the current source [1].
[1] https://github.com/wso2/carbon-apimgt/blob/release-1.10.x/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/listing/js/samples.js#L25